src/utils/common.ts

```diff
@@ -0,0 +1,58 @@
+import { parse } from "qs";
+import sanitizeHtml from "sanitize-html";
+
+export function toFixedIfNecessary(value: string, dp: number) {
+  return +parseFloat(value).toFixed(dp);
+}
+
+export function capitalizeFirstLetter(string: string) {
```

Comment:
Isn't this a css transform ?

Reply:
we have a use case where we capitalise a specific words in a sentence. css transform doesn't work in that case.